spec/models/merchant_spec.rb

```diff
@@ -73,7 +73,7 @@
 
       order_3.item_orders.create!(item: ball, price: ball.price, quantity: 1)
 
-      expect(@meg.merchant_orders).to eq([order_1.id, order_2.id])
+      expect(@meg.merchant_orders.sort).to eq([order_1.id, order_2.id].sort)
```

Comment:
Was getting this every other run:

```
 1) Merchant instance methods merchant_orders
     Failure/Error: expect(@meg.merchant_orders).to eq([order_2.id, order_1.id])
     
       expected: [3083, 3082]
            got: [3082, 3083]
```

Reply:
This seems weird because I've run the spec with this test included a lot of times and never had an error. I wonder why 🧐

Reply:
In my test it reads `expect(@meg.merchant_orders).to eq([order_1.id, order_2.id])` but in your fail message the orders in the array are reversed. In the test code above though they are correct. Could this be the problem? I have the same code and have never had an error. 🤷‍♀️

Reply:
It was more of a safe net. What I was running into to was when I ran the spec every other time it would fail so this doesnt effect anything but rather is there if something does go wrong.